KPI/dashboards/desktop_kpis_with_filters_alternate.dashboard.lookml

```diff
@@ -375,7 +374,7 @@
               From Target New Profiles Count}], showLabels: true, showValues: true,
         unpinAxis: false, tickDensity: default, tickDensityCustom: 5, type: linear}]
     reference_lines: [{reference_type: line, range_start: max, range_end: min, margin_top: deviation,
-        margin_value: mean, margin_bottom: deviation, label_position: left, color: "#1A73E8",
+        margin_value: mean, margin_bottom: deviation, label_position: center, color: "#000000",
```

Comment:
All the lines are now black and the labels are in the center (for both `At Forecast` and `Moving Avg`). I did this to make it more obvious what was/wasn't a reference line.